hrpsys_ros_bridge_tutorials/models/gen_sensor_attached_hrp2w_model.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
```

Comment:
add 5th argument to pass the name of the robot and add if sentence to gen_sensor_attached_hrp2_model.sh
https://github.com/start-jsk/rtmros_tutorials/blob/master/hrpsys_ros_bridge_tutorials/models/gen_sensor_attached_hrp2_model.sh#L6